doctoshotgun.py

```diff
@@ -547,54 +547,74 @@ def try_to_book_place(self, profile_id, motive_id, practice_id, agenda_ids, vac_
 
         return self.page.doc['confirmed']
 
+class Vaccine:
+    def __init__(self, key, motive):
+        self.key = key
+        self.motive = motive
+class Pfizer(Vaccine):
+    def __init__(self, key, motive):
+        super().__init__(key, motive)
+class Moderna(Vaccine):
+    def __init__(self, key, motive):
+        super().__init__(key, motive)
+class Janssen(Vaccine):
+    def __init__(self, key, motive):
+        super().__init__(key, motive)
+class AstraZeneca(Vaccine):
+    def __init__(self, key, motive):
+        super().__init__(key, motive)
```

Comment:
Is it really useful to declare classes for each vaccine?

Also, it is not required to redefine `__init__` in subclasses if you only call the parent method.

Reply:
Removed additional subclasses and main class so that name is an attribute